hanadb_exporter/utils.py

```diff
@@ -72,3 +73,7 @@ def systemd_ready():
         addr = '\0' + addr[1:]
     soc.connect(addr)
     soc.sendall(b'READY=1')
+
+
+def get_hostname():
```

Comment:
Do we need this additional method? Couldn't we just use the `gethostname` wherever we need it?

Reply:
I found `socket` is only used here in this utility, so I just preferred to keep it enclosed here and other classed can call the function, let me know if this doesn't make sense